go/flags/endtoend/vttablet.txt

```diff
@@ -748,7 +746,7 @@ Usage of vttablet:
   --security_policy string
 	the name of a registered security policy to use for controlling access to URLs - empty means allow all for anyone (built-in policies: deny-all, read-only)
   --service_map value
-	comma separated list of services to enable (or disable if prefixed with '-') Example: grpc-vtworker
+	comma separated list of services to enable (or disable if prefixed with '-') Example: grpc-vtgateservice
```

Comment:
can we change this example to be a vttablet appropriate one? like `grpc-queryservice`

Reply:
hmm.. this is coming from servenv/service_map.go. If we change it there, it changes for all binaries.